packages/richtext-lexical/src/field/index.tsx

```diff
@@ -48,10 +48,6 @@ export const RichTextField: React.FC<LexicalRichTextFieldProps> = (props) => {
       ) // Execute the clientFeatureProvider function here, as the server cannot execute functions imported from use client files
     }
 
-    const finalLexicalEditorConfig = lexicalEditorConfig
```

Comment:
We already fall back to defaultEditorLexicalConfig when destructuring the component props. => unnecessary to do that here